spark/v3.3/spark-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4

```diff
@@ -74,6 +74,7 @@ statement
     | ALTER TABLE multipartIdentifier SET IDENTIFIER_KW FIELDS fieldList                    #setIdentifierFields
     | ALTER TABLE multipartIdentifier DROP IDENTIFIER_KW FIELDS fieldList                   #dropIdentifierFields
     | ALTER TABLE multipartIdentifier createReplaceBranchClause   #createOrReplaceBranch
+    | ALTER TABLE multipartIdentifier DROP BRANCH identifier   #dropBranch
```

Comment:
can we make the `#dropBranch` in one row like `#setIdentifierFields` and `#call`
```suggestion
    | ALTER TABLE multipartIdentifier DROP IDENTIFIER_KW FIELDS fieldList                   #dropIdentifierFields
    | ALTER TABLE multipartIdentifier createReplaceBranchClause                                     #createOrReplaceBranch
    | ALTER TABLE multipartIdentifier DROP BRANCH identifier                                          #dropBranch
```

Reply:
I think it's how the diff is rendered https://github.com/apache/iceberg/blob/master/spark/v3.3/spark-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4#L74 setIdentifier/call and the others are all on the same row.

Reply:
Oh I see what you mean, did you mean aligned on the same column as the others for readability? updated

Reply:
oops yes same column